templates/zerver/emails/find_team/find_team_email.txt

```diff
@@ -0,0 +1,8 @@
+Hello {{ user_profile.name }},
+
+You can log into Zulip using {{ user_profile.email }} on following realm:
```

Comment:
We don't use the term `realm` in user-facing materials; instead it should be called an "organization".